frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -72,6 +73,9 @@ const Metric = (props: MetricProps) => {
         (delta !== undefined) && mode.push("delta");
         return [
             {
+                title: {
+                  text: props.title
```

Comment:
What is there's no title? See implementation in Chart.tsx

Reply:
I've refactored it base on the Chart.tsx